src/osha/oira/client/browser/certificate.py

```diff
@@ -189,6 +189,8 @@ def maybe_update(self):
         self.certificate.json = json.dumps(values)
 
     def __call__(self):
+        if not self.webhelpers.can_view_session:
+            return self.request.response.redirect(self.webhelpers.client_url)
```

Comment:
IIRC the certificates can be shared to the public

Reply:
That's done in the `PublicCertificate` view. https://github.com/euphorie/osha.oira/blob/cda0f254a4230e1d51e2aa920886e978fbf34976/src/osha/oira/client/browser/certificate.py#L199